controlplane/src/core/repositories/FederatedGraphRepository.ts

```diff
@@ -294,6 +298,24 @@ export class FederatedGraphRepository {
       .where(and(eq(targets.id, targetId), eq(schema.targets.organizationId, this.organizationId)));
   }
 
+  // We do not need to do any recomposition since we move both the graph and subgraph to the new namespace.
+  // Also, due to unique labels, they will not compose with other graphs.
+  // The namespaceId column update is enough.
+  public async moveMonograph({
```

Comment:
As described earlier, we need to find a better abstraction in the code base to not shot our self in the foot. Monographs are federated graphs with just one subgraph but users will work with them as a monograph.  We're trying to merge them somehow inn the codebase without clear boundaries. There are some differences when dealing with both types and this needs to be respected. I'd recommend to separate them on the service and repository level where possible. You can put these new methods in a `MonographRepository`